wordlist.txt

```diff
@@ -13,11 +13,13 @@ Furo
 Grafana
 IAM
 installable
+jq
```

Comment:
We should be able to remove jq from the wordlist if we put the instances where it appears into inline code. Sometimes I forget we changed from markdown to rst, and that rst needs double backticks (single renders in italics, not code).

Reply:
```suggestion
```